ts/navigation/ProfileNavigator.tsx

```diff
@@ -42,6 +43,11 @@ const ProfileStackNavigator = () => (
     initialRouteName={ROUTES.PROFILE_DATA}
```

Comment:
Should the `initialRoutName` also depend on the FF? What do you think?

Reply:
done!